lib/reports/overlap_report.rb

```diff
@@ -0,0 +1,149 @@
+require "cluster"
+require "services"
+
+Services.mongo!
+
+module Reports
+  class OverlapReport
+    # Runs an overlap report based on all the holdings of an organization
+    # and also 1+ of:
+    # * ph overlap (count other orgs w/ holdings overlap)
+    # * htdl overlap (count vols w/ overlap)
+    # * sp overlap (how many orgs have shared print overlap)
+    def initialize(organization: nil, ph: false, htdl: false, sp: false)
+      @organization = organization
+      @ph = ph
+      @htdl = htdl
+      @sp = sp
+      validate!
+    end
+
+    # Check inputs
+    def validate!
```

Comment:
I'm always leery of any code that checks a specific class in Ruby. What problems would happen if any of these were not literal `true` or `false`? I'd lean towards just doing

```ruby
def validate!
   unless [@ph, @htdl, @sp].any?
      raise "At least one of ph/htdl/sp must be true"
   end
end
```

What I suspect might be happening is concern about sidekiq & serialization, and worries about getting the string "true" or "false". I think that would be a good thing to verify in an integration test -- but not necessarily the responsibility of this class to check.

Reply:
Changed the phctl params to strings and then just checking for non-`nil`-edness.